### PR TITLE
gutlunches can be fed

### LINKED ISSUE #56059: gubbucks and guthins have stopped eating stuff again
image

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
lets you handfeed gutlunches organs and drink gibs in case they stop eating which seems to happen.
While testing locally they would eat but then stop and then start again which i can't really pinpoint why so this is a way to kind of kickstart the system.

## Why It's Good For The Game
Fixes #56059

## Changelog
:cl:
fix: fixed gutlunches annorexia
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm [hunk 1]
```diff
@@ -83,9 +83,20 @@
...
+	if(stat == CONSCIOUS)
+		if(istype(O, /obj/item/reagent_containers/glass))
+			if(O.reagents.has_reagent(/datum/reagent/liquidgibs, 5))
+				udder.generateMilk()
+				visible_message("<span class='notice'>[src] drinks from [O].</span>")
+				O.reagents.remove_reagent(/datum/reagent/liquidgibs, 5)
+			else
+				udder.milkAnimal(O, user)
+			regenerate_icons()
+		if(is_type_in_typecache(O,wanted_objects))
```
Comment (coiax) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm:95]:
```suggestion
		if(is_type_in_typecache(O, wanted_objects))
```

**Comment by Fikou** [comment]
this should use the already existing feeding system

**Comment by cacogen** [comment]
Did you try attacking them to get them to eat?

**Review by Cyberboss — CHANGES_REQUESTED** [review]
What @Fikou said

**Comment by Tlaltecuhtli** [comment]
wait what, they are supposed to drink gibs from floor and then u milk them with the glass, not hand feeding them

**Comment by cacogen** [comment]
I haven't played ash walker in a long time but it used to be you'd have to damage their health to get them to eat. Maybe it's changed and this is a genuine bug this is a workaround for.

**Comment by spessbro** [comment]
yeet

**Pull request closed without merging**